src/app/projects/projects.scss

```diff
@@ -0,0 +1,16 @@
+body {
+  .projects-container {
+    padding: 64px 0;
+  }
+  .card {
```

Comment:
Can we put spaces between these selectors?

```suggestion
body {
  .projects-container {
    padding: 64px 0;
  }

  .card {
    height: 100%;
    display: flex;
    flex-direction: column;
  }

  .card-media {
    padding: 56.25% 0 0 0;
  }
  
  .card-content {
    flex-grow: 1;
  }
}
```

Reply:
Same with all scss files/selectors.

Reply:
Okay, I will do that.